scripts/get_all_etags.py

```diff
@@ -40,5 +40,5 @@ def fetch_biosample_ids():
     main()
     today = date.today().strftime('%Y-%m-%d')
     with open("etag_list_{}.txt".format(today), 'w') as w:
-        for item in ETAG:
+        for item in sorted(ETAG):
```

Comment:
Is it crucial to have ETAGs sorted?

Reply:
I would say 100% for debug purpose. No order, then diff yesterday_list today_list will result in a long list of difference even though there is no update.